src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -155,9 +156,9 @@ public async Task ExecuteAsync(ExportJobRecord exportJobRecord, WeakETag weakETa
                     // Best effort to delete the secret. If it fails to delete, then move on.
                     await _secretStore.DeleteSecretAsync(_exportJobRecord.SecretName, cancellationToken);
                 }
-                catch (Exception ex)
+                catch (SecretStoreException sse)
```

Comment:
nit: I think this should remain Exception since what we want is fire and forget.